src/core/Accordion.tsx

```diff
@@ -74,32 +81,42 @@ const AccordionRow = ({
   const [contentHeight, setContentHeight] = useState<number>(0);
 
   useEffect(() => {
-    const handleHeight = throttle(() => {
+    const resizeObserver = new ResizeObserver(() => {
       if (rowRef.current) {
         setContentHeight(rowRef.current.scrollHeight + 16);
       }
-    }, 250);
+    });
 
-    handleHeight();
+    if (rowRef.current) {
+      resizeObserver.observe(rowRef.current);
```

Comment:
This is a superior api for judging resize events as it also takes into account the resizing of the target element, not just the window. We need this for accordions-inside-of-accordions, as pricing demands